src/tests/unit/data/documentSelection-events.ts

```diff
@@ -84,6 +87,227 @@ export const INVALID_SESSION_ID = {
 export const MISSING_SESSION_ID = {
 	...VALID_REQUEST,
 	headers: {},
+}
+;
+export const MISSING_PDF_PREFERENCE = {
+	...VALID_REQUEST,
+	body: JSON.stringify({
+		"document_selection":{
+			"document_selected":"ukPassport",
+			"date_of_expiry":"1970-01-01",
+		},
+		"post_office_selection":{
+			"address":"1 The Street, Funkytown",
+			"location":{
+				"latitude":0.34322,
+				"longitude":-42.48372,
+			},
+			"post_code":"SW19 4NS",
+		},
+	}),
+};
+
+export const MISSING_UPRN = {
+	...VALID_REQUEST,
+	body: JSON.stringify({
+		"document_selection":{
+			"document_selected":"ukPassport",
+			"date_of_expiry":"1970-01-01",
+		},
+		"post_office_selection":{
+			"address":"1 The Street, Funkytown",
+			"location":{
+				"latitude":0.34322,
+				"longitude":-42.48372,
+			},
+			"post_code":"SW19 4NS",
+		},
+		"pdf_preference": "PRINTED_LETTER",
```

Comment:
Can we use the enum instead of string here? Easy to update/trace for any future changes.